sdk/core/core-tracing/review/core-tracing.api.md

```diff
@@ -75,15 +86,16 @@ export { OpenCensusSpan }
 // @public
 export class OpenCensusSpanWrapper implements Span {
     constructor(span: OpenCensusSpan);
-    constructor(tracer: OpenCensusTracerWrapper, name: string, options?: OTSpanOptions);
-    addEvent(_name: string, _attributes?: Attributes): this;
+    constructor(tracer: OpenCensusTracerWrapper, name: string, options?: OTSpanOptions, context?: Context_2);
```

Comment:
feels a little weird to have two definitions for `Context`

Reply:
Fixed!